test/service/test_logging.py

```diff
@@ -0,0 +1,54 @@
+import json
+import re
+import requests
+
+from service import WebServiceTestCase
+
+from azul.chalice import log
+from azul.logging import configure_test_logging
+
+
+def setUpModule():
+    configure_test_logging()
+
+
+class TestRequestResponse(WebServiceTestCase):
```

Comment:
Same as in the other PR. The ES container that `WebServiceTestCase` provides is not necessary for this test.